pagespeed/system/system_test.sh

```diff
@@ -2143,6 +2143,7 @@ function cache_purge_test() {
       echo $CURL --request PURGE --proxy $SECONDARY_HOSTNAME "$PURGE_URL"
       check $CURL --request PURGE --proxy $SECONDARY_HOSTNAME "$PURGE_URL"
     fi
+    echo ""
```

Comment:
actually, no, this was on purpose.  For years I've been bothered a little bit by this message in the system-test logs:
    Purge successfulsleep 6
this brilliant bit of scripting makes it
    Purge successful
    sleep 6